playbooks/openstack/openshift-cluster/install.yml

```diff
@@ -8,8 +8,8 @@
 # values here. We do it in the OSEv3 group vars. Do we need to add
 # some logic here?
 
-- name: run the cluster deploy
-  import_playbook: ../../prerequisites.yml
+- import_playbook: ../../prerequisites.yml
+- import_playbook: ../../init/cluster_facts.yml
```

Comment:
I'm unsure why this would need to be here.

Reply:
Ah you're right, that's not necessary here. I'll remove the cluster_facts.